spec/erb_spec.rb

```diff
@@ -1,10 +1,11 @@
 require File.dirname(__FILE__) + '/spec_helper'
+require 'redcloth/erb_extension'
```

Comment:
Any idea how to do that? To preserve the same functionality (auto-inclustion to the ERB helpers), I believe it's impossible to do. This could be changed to use another module, include it into a `ERB::Utils` and then remove module later when test unit is completed. Let me know your thoughts.